src/crowdsale/StakedLockingCrowdSale.sol

```diff
@@ -27,7 +27,8 @@ error BadPrice();
 /**
  * @title StakedLockingCrowdSale
  * @author molecule.to
- * @notice a fixed price sales base contract that locks the sold tokens in a configured locking contract and requires vesting another ("dao") token for a certain period of time to participate
+ * @notice a fixed price sales contract that locks the sold tokens in a configured locking contract and requires vesting another ("dao") token for a certain period of time to participate
+ * @dev see https://github.com/moleculeprotocol/IPNFT
```

Comment:
this comment just leads to this repo. I don't understand how this helps?

Reply:
You can click on it on etherscan?